Docs/sphinx/Tutorials.rst

```diff
@@ -67,13 +69,33 @@ Update ``poetry``::
 
 Make sure the list of non-QSS species is correct in ``Support/Mechanism/Models/dodecane_lu_qss/non_qssa_list.yaml``
 
-Generate ``qssa.yaml``::
+In the next step, ``skeletal.yaml`` is the ``mechanism.yaml`` of the skeletal version of the mechanism (here available under ``${PELE_PHYSICS_HOME}/Support/Mechanism/Models/dodecane_lu``).
+If only CHEMKIN files are available for the skeletal mechanism, see :ref:`Converting CHEMKIN files <sec_convertCK>` for generating ``skeletal.yaml``.
+
+Generate ``qssa.yaml`` from ``skeletal.yaml`` and ``non_qssa_list.yaml``::
 
   $ cd ${PELE_PHYSICS_HOME}/Support/ceptr
   $ poetry run qssa -f ${PELE_PHYSICS_HOME}/Support/Mechanism/Models/dodecane_lu_qss/skeletal.yaml -n ${PELE_PHYSICS_HOME}/Support/Mechanism/Models/dodecane_lu_qss/non_qssa_list.yaml
 
-Generate ``mechanism.H`` and ``mechanism.cpp``::
+Generate ``mechanism.H`` and ``mechanism.cpp`` from ``qssa.yaml``::
   
   $ cd ${PELE_PHYSICS_HOME}/Support/ceptr
   $ poetry run convert -f ${PELE_PHYSICS_HOME}/Support/Mechanism/Models/dodecane_lu_qss/qssa.yaml
 
+Tutorial 3 - Generating NC12H26 Skeletal mechanism
+-------------------------------------------------------------------------
```

Comment:
this extra long line will probably generate a warning on the docs, can you make it fit?